src/decode-template.c

```diff
@@ -62,6 +62,9 @@ int DecodeTEMPLATE(ThreadVars *tv, DecodeThreadVars *dtv, Packet *p,
         //ENGINE_SET_EVENT(p,TEMPLATE_HEADER_TOO_SMALL);
         return TM_ECODE_FAILED;
     }
+    if (!PacketIncreaseCheckLayers(p)) {
```

Comment:
A comment explaining what is happening would be good here. The template should be verbose about such things.

Reply:
ok